.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,59 +1,60 @@
+#### ~ 1 day before the release date.
+
+- [ ] Open a PR in the k6-docs repository, archiving the current k6's JavaScript API version as per the following [instructions](https://github.com/grafana/k6-docs/wiki/Add-version-for-Javascript-API-documentation).
+- [ ] Ensure the [existing k6-docs PRs](https://github.com/grafana/k6-docs/pulls), related to the new functionalities and changes, are reviewed, up to date with the latest state of the `master` branch, and based upon the branch containing the k6 archived JavaScript API documentation (as created in the previous step).
 
-- [ ] PR for archiving the current k6's JavaScript [API version](https://github.com/grafana/k6-docs/wiki/Add-version-for-Javascript-API-documentation).
-- [ ] Check that the [Existing k6-docs PRs](https://github.com/grafana/k6-docs/pulls) related to the new functionality are reviewed and rebased and pointing to the branch with k6's JavaScript API archived.
- 
 ### Release Day
 
 #### Documentation
 
-- [ ] The PR with archiving the old version JS API merged first and rebase the rest on top.
-- [ ] PRs with changes related to the release merged.
-- [ ] After merging all k6-docs' PRs ensure that we have no broken links by checking "Check broken links" job in GitHub actions. 
-- [ ] [The new Docs Release vX.Y.Z](https://github.com/grafana/k6-docs/releases/new) published.
-- [ ] Release Notes PR contains the right links to the docs.
+- [ ] Merge the k6-docs repository's Javascript API archiving PR and rebase the rest of the branches meant for the release on top of the new state of the `master` branch.
+- [ ] Merge all the k6-docs repository's branches containing changes related to the release.
+- [ ] Ensure the last resulting k6-docs GitHub action targetting the `main` branch sees its "Check broken links" job pass.
+- [ ] Publish the new [vX.Y.Z version of docs](https://github.com/grafana/k6-docs/releases/new).
+- [ ] Ensure the k6 repository release notes PR contains the correct links to the docs.
 
 #### In k6 repository
 
-- [ ] PR for bumping [the version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11) merged.
-- [ ] Release notes PR merged.
-- [ ] A new tag from the CLI `vX.Y.Z` created (`git tag v0.4x.0 -m "v0.4x.0"`) & pushed.
+- [ ] Merge the PR bumping [the k6 Go project's version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11).
+- [ ] Merge the release notes PR.
+- [ ] Create and push a new tag of the form `vX.Y.Z` using git: `git tag v0.4x.0 -m "v0.4x.0"`.
 
 #### Announcements
 
-- [ ] A GitHub's link to the new release published in #k6-changelog.
-- [ ] DevRel team is notified in #k6-devrel that release is published.
-- [ ] The release's milestone closed.
+- [ ] Publish a link to the new GitHub release in the #k6-changelog channel.
+- [ ] Notify the DevRel team in the #k6-devrel channel, letting them know that the release is published.
```

Comment:
Same comment about as above -this is internal things. I wonder also if we shouldn't automate them as well :thinking: 

But I guess they have been around for a while so not that big of a problem

Reply:
I think we should also simplify things here, we should only post it in `#k6` because it is the channel everyone should follow after the generals.